.github/workflows/go.yml

```diff
@@ -0,0 +1,34 @@
+name: Go
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build:
+    strategy:
+      matrix: 
+        go-version: [1.12.x, 1.13.x, 1.14.x]
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go-versions }}
+    - name: Run gofmt
+      run: diff -u <(echo -n) <(gofmt -d *.go)
+    - name: Run golint
+      run: diff -u <(echo -n) <(golint $(go list -e ./...) | grep -v YAMLToJSON)
+    - name: Run go vet
+      run: GO111MODULE=on go vet .
+    - name: Run go test
+      run: GO111MODULE=on go test -v -race ./...
+    - name: Check diff
+      run: git diff --exit-code
+    - name: Install
+      run: GO111MODULE=off go get golang.org/x/lint/golint
```

Comment:
This is a dependency of the whole process. Please move it to right after `Setup Go`.

Reply:
Saw the wierdness that linter installation came after linting...and now on second thought should have flagged it ... (first pr ever so did not want to question as this seemed like a working sequence. Now seems like travis has may have special treatment for the name "Install". Moved to beginning now